src/txn/txn.c

```diff
@@ -926,8 +926,6 @@ __txn_resolve_prepared_op(WT_SESSION_IMPL *session, WT_TXN_OP *op, bool commit,
          * our snapshot in prepare. Otherwise, we can't see updates with non-globally visible
          * transaction ids.
          */
-        F_SET(hs_cursor, WT_CURSTD_IGNORE_TOMBSTONE);
```

Comment:
Shall we remove the comment as well?

Reply:
Good point, I think only the first sentence is relevant now.

Reply:
1608889